File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem04_CanPlaceFlowersTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertTrue;

class Problem04_CanPlaceFlowersTest {

    private final Problem04_CanPlaceFlowers test = new Problem04_CanPlaceFlowers();

    @Test
    void canPlaceFlowers() {
        int[] flowerbed = new int[]{1,0,0,0,1};
        assertTrue(test.canPlaceFlowers(flowerbed,1));
    }
}

File: JavaScript/leetcode75/src/test/Problem04_CanPlaceFlowers.test.js
const canPlaceFlowers = require("../main/Problem04_CanPlaceFlowers");

test("should place flowers", () => {
  const flowerbed = [1, 0, 0, 0, 1];
  expect(canPlaceFlowers(flowerbed, 1)).toBe(true);
});
